<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>liveGUI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/icons.qrc">
    <normaloff>:/icons/icons/livestream-ICON.png</normaloff>:/icons/icons/livestream-ICON.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="topMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabBar::tab { width: 150px; }</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="iconSize">
       <size>
        <width>70</width>
        <height>70</height>
       </size>
      </property>
      <widget class="QWidget" name="browseTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/remote-control.png</normaloff>:/icons/icons/remote-control.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="BrowsingWidget" name="browsingWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="favoritesTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/browse-favs.png</normaloff>:/icons/icons/browse-favs.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>9</number>
        </property>
        <property name="topMargin">
         <number>9</number>
        </property>
        <property name="rightMargin">
         <number>9</number>
        </property>
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="FavoriteWidget" name="favoriteWidget" native="true"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="previewTab">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/preview.png</normaloff>:/icons/icons/preview.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QScrollArea" name="previewScroll">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>588</width>
             <height>333</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="PreviewStreamWidget" name="previewStreamWidget" native="true"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="settingsTab">
       <attribute name="icon">
        <iconset resource="../qrc/icons.qrc">
         <normaloff>:/icons/icons/remoteAccess.png</normaloff>:/icons/icons/remoteAccess.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>FavoriteWidget</class>
   <extends>QWidget</extends>
   <header>favoritewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PreviewStreamWidget</class>
   <extends>QWidget</extends>
   <header>previewstreamwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BrowsingWidget</class>
   <extends>QWidget</extends>
   <header>browsingwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qrc/icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>updateLivestreamOutput(QString)</signal>
 </slots>
</ui>
